CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## Solidus (not yet versioned)
+
+*   Address is immutable (Address#readonly? is always true)
+
+    This allows us to minimize cloning addresses, while still ensuring historical
+    data is preserved.
+
+*   PUT /api/addresses without an order no longer works
```

Comment:
Right. So you think this line should be removed from the changelog?

Sure, I'll give a shout out to admin searches